workspaces/local-cli/src/commands/login.ts

```diff
@@ -26,19 +27,19 @@ export default class Login extends Command {
       const queryString = qs.stringify({
         tokenUrl: `http://localhost:${port}/api/token`
       });
-      const parsedUrl = url.parse(`${loginBaseUrl}/login`);
-      parsedUrl.query = queryString;
-      const launchUrl = url.format(parsedUrl);
```

Comment:
this was just missing the query option https://nodejs.org/api/url.html#url_url_format_url_options